programs/rol_stack.zasm

```diff
@@ -0,0 +1,17 @@
+	.text
+	.file	"rol_stack.zasm"
+	.globl	__entry
+__entry:
+.func_begin0:
+	; grow stack
+	add 1, r0, stack+=[2]
+	; set stack value
+	add 1, r0, stack=[0]
```

Comment:
Right now stack=[] is not working correctly, it is interpreted as stack[] , in order to avoid future errors if this changes, please change all stack=[] for stack[]